src/components/NewLocationPage/NotesBlock/NotesBlock.style.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const TextContainer = styled.div`
+  display: inline-block;
+  flex-direction: column;
+  font-weight: bold;
+`;
+
+export const TextComponent = styled.div`
```

Comment:
we want text to render in the dom as text. making this a `p` element instead of a `div` would do the trick